src/include/cart/api.h

```diff
@@ -1909,6 +1889,64 @@ int crt_group_view_destroy(crt_group_t *grp);
  */
 int crt_group_psr_set(crt_group_t *grp, d_rank_t rank);
 
+/**
+ * Add rank to the specified primary group.
+ *
+ * Passed ctx will be used to determine a provider for which the uri is
+ * being added in the case of the primary group.
+ *
+ * For primary groups when uri is specified, the uri is assumed
+ * to be 'base' URI, corresponding to tag=0 of the node being added.
+ *
+ * \param[in] ctx               Associated cart context
+ * \param[in] grp               Group handle
+ * \param[in] primary_rank      Primary rank to be added
+ * \param[in] uri		URI of the primary rank
```

Comment:
There's something odd with alignment here.

Reply:
Yep, tabs were used on 1904 instead of spaces elsewhere. will fix